test/functional/apps/discover/index.js

```diff
@@ -19,6 +19,7 @@ define(function (require) {
     bdd.before(function () {
       common = new Common(this.remote);
       remote = this.remote;
+      this.timeout = 120000;
```

Comment:
Does this need to be in the `before`? I think if you put it in the top level `describe` it should apply to all tests inside it (which, is all Discover tests).